packages/core/tsconfig.json

```diff
@@ -1,7 +1,12 @@
 {
-  "extends": "../tsconfig.json",
+  "extends": "@posthog-tooling/tsconfig-base",
```

Comment:
What is `@posthog-tooling` is that a separate package or something that lives inside the same repo?